.vscode/settings.json

```diff
@@ -195,7 +195,6 @@
     "**.pb.go": true,
     "**/*.gen.json": true,
     "**/testdata/*": true,
-    "**Generated.ts": true,
```

Comment:
this settings affects the quick open menu, which I use heavily to navigate the codebase, and which means I have to go to great lengths just to open typesGenerated.ts. I quite dislike these opinionated sorts of configs being baked into the codebase. 😅

Reply:
the opposite; it was being ignored before, and now will actually show up.